### DISCUSSION TITLE
multi views and switches in min browser?

### DISCUSSION THREAD

**Discussion by ghost** [minbrowser/min#2152] [Q&A] [answered]
Hello everyone!

The next feature I plan on releasing in the min browser is the view switch feature. *So... my idea is to bring  [Bonsai browser](https://github.com/Bonsai-Desk/bonsai-browser) features and features in launchpad in mac-os to Min browser.*

#### Idea: img 1 - grid view in min browser
As we can see in the image there is a **grid-view** for list of access sites or a list of sites that are going to be access. *This img is licensed from [css-tricks.com/bonsai-browser](https://css-tricks.com/bonsai-browser/)*

![image](https://user-images.githubusercontent.com/111701513/211178365-12631e77-c55e-4838-bc70-f3d48ec02636.png)

#### Idea: img 2 - taskview in min browser *default*
As we can see in the image there is a **task-view** for list of access sites or a list of sites that are going to be access. *This img is licensed from min browser in homepage*

![image](https://user-images.githubusercontent.com/111701513/211178686-7bc26e04-f492-4ada-acd4-bca5ad5651a8.png)

#### Idea: img 3 - launchpad view in min browser
As we can see in the image there is a **launchpad** for list of access sites or a list of sites that are going to be access - this img is licensed from [help.apple.com/assets](https://help.apple.com/assets/). *Every website I visit has a favicon that becomes an app image like launchpad in macOS in min browser.*

![image](https://user-images.githubusercontent.com/111701513/211178620-a96df94d-b879-4235-813c-44360febb545.png)

#### Idea: img 4 - vertical-tabs view in min browser
*This img is licensed from [aldo-cervantes](https://www.flaticon.com/br/autores/aldo-cervantes)*

![image](https://user-images.githubusercontent.com/111701513/211183687-06ddc87d-ae88-49bf-a4ef-5a3bc47d9e36.png)

#### Idea: video
Please, watch the video: https://videopress.com/embed/xzbefI74

#### why?
1. It's a minimalist design, which is in line with Min Browser proposal
2. Have a lot of tab is a waste of time *(In my opinion)*
3. A good part of users use cell phones, having a design similar to cell phones like the grid/list I mentioned here can be interesting to have a good part of users use cell phones in Browser Min
4. Bonsai Browser is a project that is no longer active, but has interesting features for the Browser Min
5. [opera-neon](https://blogs.opera.com/desktop/2017/01/opera-neon-concept-browser/) have this concept
6. Min Browser can alternative to opera-neon, [Bonsai Browser](https://bonsaibrowser.com/)
7. Some users of opera-neon, bonsai browser can use Min browser because it is similar to these solutions
8. my idea is to bring Bonsai browser features to Min browser.

#### question/feedback
What do you all think of the idea?

**Comment by PalmerAL** [comment, marked as answer]
Would this work the same way as the existing tasks feature, but with a grid view, or does it work differently?

**Reply by ghost** [reply]
Hi PalmerAL.

> Would this work the same way as the existing tasks feature, but with a grid view, or does it work differently?

but with a grid view *(Yes)*

**Reply by PalmerAL** [reply]
OK, yeah, adding a grid/list view toggle to the task overlay sounds reasonable to me. We have an existing property on the tab object that stores a base64-encoded thumbnail of the tab that you could use to implement this: https://github.com/minbrowser/min/blob/master/js/tabState/tab.js#L29

**Reply by ghost** [reply]
Hi PalmerAL.

> OK, yeah, adding a grid/list view toggle to the task overlay sounds reasonable to me. We have an existing property on the tab object that stores a base64-encoded thumbnail of the tab that you could use to implement this: https://github.com/minbrowser/min/blob/master/js/tabState/tab.js#L29

Thank you for feedback.
